specification/offazurespringboot/resource-manager/Microsoft.OffAzureSpringBoot/preview/2023-01-01-preview/springbootdiscovery.json

```diff
@@ -1228,18 +1319,19 @@
     },
     "springbootsitesPatch": {
       "type": "object",
-      "allOf": [
-        {
```

Comment:
>tags [](http://example.com/codeflow?start=9&length=4)

Please switch back to the TrackedResource definition from v3 common types. That definition has the correct mutability, and you don't need to add your own top-level properties here.

Reply:
For this object patch of resource(springbootsites), if we reference the trackedresource, we must add required field, but for the swagger validation it can't include required filed, so we remove the reference and add tags on the top level

Reply:
OK, Just a recommendation I thought should work. It's fine to leave it as-is.